src/library/scala/collection/mutable/LongMap.scala

```diff
@@ -338,8 +338,11 @@ final class LongMap[V] private[collection] (defaultEntry: Long => V, initialBuff
   }
 
   /** Adds a new key/value pair to this map and returns the map. */
+  @deprecated("Use `+= ((k, v))` instead of `+= (k, v)`", "2.13.0")
   def +=(key: Long, value: V): this.type = { update(key, value); this }
 
+  @inline override final def +=(kv: (Long, V)): this.type = { update(kv._1, kv._2); this }
```

Comment:
Doesn't this already forward to the `addOne` below?

Reply:
Yes, it forwards to the same method (in dynamic dispatch) but it doesn't inline the overridden method where the concrete types are known so that the optimizer can unbox.